views/docs.handlebars

```diff
@@ -1,4 +1,21 @@
 <main role="main">
-    <div class="container center-content">
+    <div class="docs-header container center-content">
+        <h1>Documentation</h1>
+        <div class="docs">
```

Comment:
Add a table of contents like things to each of the different sections on this page because each one is long. As a user I don't want to have to scroll through everything looking for the one piece I am interested in,

Reply:
Yes, i plan to improve the UI in the next few weeks. Just wanna go for minimum viable product for now

Reply:
okay that works. A new issue should be opened after this is done.

Reply:
Will do

Reply:
See #109